resolwe_runtime_utils.py

```diff
@@ -15,25 +15,114 @@
 """
 Utility functions that make it easier to write a Resolwe process.
 """
+import functools
 import glob
 import gzip
 import json
+import logging
 import os
 import re
 import shlex
 import shutil
+import socket
 import subprocess
 import tarfile
 import zlib
+import time
+from contextlib import suppress
+from itertools import chain
+from pathlib import Path
 
-import requests
+
+# Socket constants. The timeout is set to infinite.
+SOCKET_TIMEOUT = None
+if "SOCKET_TIMEOUT" is os.environ:
+    SOCKET_TIMEOUT = int(os.environ["SOCKET_TIMEOUT"])
+SOCKETS_PATH = Path(os.environ.get("SOCKETS_VOLUME", "/sockets"))
```

Comment:
It would be nice to raise a user-friendly message if environment variables are not defined.

Reply:
There are sensible defaults for them. Should I really raise exception if they are not defined?